Update "Publication Policies"

aguilerapy opened the pull request:
Change the name from "Publication Policies" to "Publication Policies: Data User Guide".

Based on CRM issue 5167.

aguilerapy committed:
Change "Publication Policies" name

Change the name from "Publication Policies" to "Publication Policies: Data User Guide".

jpmckinney commented:
@aguilerapy This will make the menu item in the left-hand sidebar very long (maybe longer than the sidebar): https://standard.open-contracting.org/latest/en/implementation/publication_policy/

Also, I think we'd want to edit some of the text on the page to more consistently use 'data user guide' instead of (or along with) publication polic*.

aguilerapy commented:
![2019-10-18-140152_1121x670_scrot](https://user-images.githubusercontent.com/48607824/67113519-162b8600-f1b0-11e9-9cde-03139c947405.png)

@jpmckinney The left-hand sidebar don't change, the text "Guide" ran to the second line. 

![2019-10-18-140210_998x678_scrot](https://user-images.githubusercontent.com/48607824/67113800-c8634d80-f1b0-11e9-96b8-78dc4cdee137.png)

We can also leave "Data User Guide" as subtitle in publication_policy.md what do you think?

jpmckinney commented:
Yes, my concern is that it goes onto two lines, whereas we try to keep each menu item on one line.

The subtitle will unfortunately create an extra layer of hierarchy, which isn't something we want.

I think we should focus on editing the text to make it clearer that this policy should function as a data user guide (we can bold those words). 

We should work to align the content from the newer publication policy guidance ([EN](https://www.open-contracting.org/resources/ocds-1-1-publication-policy-template/), [ES](https://www.open-contracting.org/resources/ocds-1-1-plantilla-de-politica-de-publicacion-es/)) with this page.

That would be a bigger project, but it will have more impact than adding a few words to this page.

aguilerapy commented:
Okay, I will work on that

aguilerapy committed:
Update publication_policy.md

Update the content from the newer publication policy guidance template with this publication_policy.md

aguilerapy commented on standard/docs/en/implementation/publication_policy.md:
```diff
@@ -1,17 +1,26 @@
+7. Details of the license under which the data is published.
+8. Any relevant additional documentation that the reader should be aware of when using a publishers OCDS dataset.
+9. Future development plans - brief notes on the future plans of the publisher to improve their data.
+10. The limits of the publishers legal responsibilities for the data.
```
Are the summary sections that are in the template drive document.

aguilerapy committed:
Update publication_policy.md

jpmckinney commented:
@aguilerapy Let me know when there is a new version ready for review (no rush).

aguilerapy commented:
@jpmckinney free to review!

jpmckinney commented:
@LindseyAM to review

LindseyAM commented:
Thanks both! I've pulled together some (even shorter) text in this google doc.

My understanding it will fall under the Publish subheading beneath the Guidance Heading in 1.1.5

James, feel free to make any edits and incorporate into documentation

https://docs.google.com/document/d/1cArKYbzYpfVt7WuIPTZz2Ot2JLahhESgc_q6SFeGSnA/edit#

jpmckinney commented:
After merging, check if #759 is closed.

jpmckinney commented:
The text from the linked document will likely be merged as part of #986. Once that's done, I can review the changes here, to see if we missed anything.

jpmckinney commented:
Superseded by #986.
